shub/apps/library/views/images.py

```diff
@@ -732,6 +734,79 @@ def get(self, request):
         # collections = generate_collections_list(token.user)
         return Response(data={}, status=200)
 
+    def post(self, request):
+        """Mimic the creation a new container.
+
+        POST /v1/coontainers
+
+        Body parameters:
+        * collection: collection numeric id as a string
+        * name: new container name
+
+        Sylabs library has an optional 'private' parameter, which we
+        ignore here because containers' privacy is inherited from the
+        collection they belong to.
+
+        Return the newly created container.
```

Comment:
I don't quite understand this endpoint - you mention that we return the newly created container, but then you mention that we "mimic the Sylabs library without actually creating the container object." What's the point?

Reply:
You're right. The comment should be fixed.

Reply:
Fixed.